src/hot-score.js

```diff
@@ -1,17 +1,19 @@
 
```

Comment:
perhaps move this file under `src/core`? or what does the masterminds @kimmobrunfeldt / @tomi think?